src/main/java/org/folio/circulation/domain/RequestServiceUtility.java

```diff
@@ -83,6 +83,22 @@ private RequestServiceUtility() {
     }
   }
 
+  static Result<RequestAndRelatedRecords> refuseWhenUserIsInactive(
+    RequestAndRelatedRecords request) {
+
+    User requester = request.getRequest().getRequester();
+    if (requester.isInactive()) {
+      Map<String, String> parameters = new HashMap<>();
```

Comment:
I think it might read more easily if there was whitespace between the sections for the validation error.

Reply:
I added some whitespace where I thought it made sense, but I'm not 100% sure if it's what you were looking for.

Reply:
Thanks, that works better for me